tox.ini

```diff
@@ -0,0 +1,34 @@
+[tox]
+envlist =
+    py{26,27,32,34,35,36,37,38,39,310}
+    flake8
+
+[gh-actions]
+python =
+    2.7: py27
+    3.5: py35
+    3.6: py36
+    3.7: py37
+    3.8: py38
+    3.9: py39
+    3.10: py310
+
+[testenv]
+deps =
+    py38: greenlet
```

Comment:
maybe we can add a comment here why `greenlet` is installed (I think it's to run tests with greenlet, at least for one of the py versions right?)

Reply:
Yes, you are right.
I've added a comment.